package.json

```diff
@@ -23,11 +23,15 @@
   },
   "homepage": "https://github.com/parasoft/run-soatest-action#readme",
   "devDependencies": {
+    "@actions/core": "^1.10.1",
```

Comment:
Should this be a dependency which needs to run in production instead of development process?